src/liblsquic/lsquic_ev_log.c

```diff
@@ -25,7 +30,7 @@
 /* Messages that do not include connection ID go above this point */
 
 #define LSQUIC_LOG_CONN_ID cid
-#define LCID(a...) LSQ_LOG2(LSQ_LOG_DEBUG, a)   /* LCID: log with CID */
+#define LCID(...) LSQ_LOG2(LSQ_LOG_DEBUG, __VA_ARGS__)   /* LCID: log with CID */
```

Comment:
This is a more portable way to do it.  I recommend that this type of change remains.